truffleruby/src/main/ruby/post-boot/post-boot.rb

```diff
@@ -51,7 +51,7 @@
       begin
         gem 'did_you_mean'
         require 'did_you_mean'
-      rescue Gem::LoadError, LoadError => e
+      rescue Gem::LoadError, LoadError => e # rubocop:disable Lint/ShadowedException
```

Comment:
The classes are equal, it wrongly reports shadowing. I am guessing the 2 classes are there for compatibility with some rubygem version.